pulp_deb/tests/functional/conftest.py

```diff
@@ -453,6 +461,35 @@ def _deb_sync_repository(remote, repo):
     return _deb_sync_repository
 
 
+@pytest.fixture(scope="class")
+def deb_copy_content(apt_copy_api, monitor_task):
+    """Fixture that copies deb content from a source repository version to a target repository."""
+
+    def _deb_copy_content(source_repo_version, dest_repo, content=None, structured=True):
+        """Copy deb content from a source repository version to a target repository.
+
+        :param source_repo_version: The repository version href from where the content is copied.
+        :dest_repo: The repository href where the content should be copied to.
+        :content: List of package hrefs that should be copied from the source. Default: None
+        :structured: Whether or not the content should be structured copied. Default: True
+        :returns: The task of the copy operation.
+        """
+        config = (
+            {"source_repo_version": source_repo_version, "dest_repo": dest_repo}
+            if content is None
+            else {
+                "source_repo_version": source_repo_version,
+                "dest_repo": dest_repo,
+                "content": content,
+            }
+        )
```

Comment:
Possibly somewhat more readable:
```suggestion
        config = {"source_repo_version": source_repo_version, "dest_repo": dest_repo}
        if content:
            config["content"] = content
```